packages/page-claims/src/Claim.tsx

```diff
@@ -1,13 +1,13 @@
 // Copyright 2017-2020 @polkadot/app-claims authors & contributors
 // SPDX-License-Identifier: Apache-2.0
 
-import { Option } from '@polkadot/types';
 import { BalanceOf, EthereumAddress, StatementKind } from '@polkadot/types/interfaces';
-
+import { TxCallback } from '@polkadot/react-components/Status/types';
 import React, { useEffect, useState } from 'react';
 import styled from 'styled-components';
+import { Option } from '@polkadot/types';
```

Comment:
This is correct (discounting the types just above that should move one up, but this one is indicated 100% as a non-type import from something called "types")